shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-common/src/main/java/org/apache/shardingsphere/sharding/rule/ShardingRule.java

```diff
@@ -326,10 +326,10 @@ public boolean isSingleTablesInSameDataSource(final Collection<String> logicTabl
     }
     
     /**
-     * Judge if there is at least one table rule for logic tables.
+     * Judge if there is at least one table rule for logic tables or not.
```

Comment:
@KSMASTERS `Judge whether a table rule exists for logic tables or not` may be better.